src/web_interface/components/miscellaneous_routes.py

```diff
@@ -78,6 +78,7 @@ def _app_delete_firmware(self, uid):
     def _app_find_missing_analyses(self):
         template_data = {
             'missing_files': self._find_missing_files(),
+            'orphaned_files': self._find_orphaned_files(),
```

Comment:
This (and also `failed_analyses`) is missing in the corresponding REST API endpoint.

Reply:
added both to REST and added tests